GitUI/Script/ScriptOptionsParser.cs

```diff
@@ -236,6 +238,10 @@ private static string CreateOption([NotNull] string option, bool quoted)
             return result;
         }
 
+        [CanBeNull]
+        private static string EscapeLinefeeds([CanBeNull] string multiLine)
```

Comment:
```suggestion
        private static string EscapeLineFeeds([CanBeNull] string multiLine)
```

Maybe turn into a local static function?

Reply:
done